anima-jws-voucher.mkd

```diff
@@ -278,14 +260,13 @@ Thanks for the supporting PoC implementations to Hong Rui Li and He Peng Jia.
 
 # Examples
 
-These examples are folded according to {{RFC8792}} Single Backslash rule.
+These examples are folded according to the {{RFC8792}} Single Backslash rule.
 
-## Example Pledge-Voucher-Request - PVR (from Pledge to Registrar)
-The following is an example request sent from a Pledge to the Registrar, in "General JWS JSON Serialization".
+## Example Pledge-Voucher-Request (PVR)
 
-~~~~
-=============== NOTE: '\' line wrapping per RFC 8792 ================
```

Comment:
This invalid comment syntax adds even more issues in my opinion. If we should duplicate the note at the top to each example, it should not be within the listing itself.